api/src/main/java/io/strimzi/api/kafka/model/kafka/KafkaClusterSpec.java

```diff
@@ -63,7 +63,7 @@ public class KafkaClusterSpec implements HasConfigurableMetrics, HasConfigurable
             + "node.id, process.roles, controller., metadata.log.dir, zookeeper.metadata.migration.enable, " // KRaft options
             + "client.quota.callback.static.kafka.admin., client.quota.callback.static.produce, client.quota.callback.static.fetch, "
             + "client.quota.callback.static.storage.per.volume.limit.min.available., client.quota.callback.static.excluded.principal.name.list, "
-            + "kafka.metric.reporters, prometheus.metrics.reporter, prometheus.metrics.reporter.listener";
+            + "kafka.metric.reporters, prometheus.metrics.reporter";
```

Comment:
You are missing a `.` here. It's `prometheus.metrics.reporter.`. The prefix is still ok, but visually it's not easy to see that this is a prefix and not a single config. It's more for us than the machine.

Reply:
Makes sense